jerry-libc/CMakeLists.txt

```diff
@@ -39,7 +39,10 @@ add_library(${JERRY_LIBC_NAME} STATIC ${SOURCE_LIBC} ${TARGET_SPECIFIC_LIBC_SOUR
 target_compile_definitions(${JERRY_LIBC_NAME} PRIVATE ${DEFINES_LIBC})
 target_include_directories(${JERRY_LIBC_NAME} PRIVATE ${INCLUDE_LIBC})
 target_include_directories(${JERRY_LIBC_NAME} SYSTEM PUBLIC "${CMAKE_SOURCE_DIR}/jerry-libc/include")
-target_link_libraries(${JERRY_LIBC_NAME} -lgcc)
+
+if(NOT USING_MSVC)
+  target_link_libraries(${JERRY_LIBC_NAME} -lgcc)
+endif()
```

Comment:
If jerry-libc gets disabled for MSVC by default, this change will become unnecessary.

Reply:
this is necessary， msvc have no gcc

Reply:
Trust me, it isn't. Jerry-libc won't work for Windows. So better disable it in the root cmakelists.txt completely for MSVC builds, and then this cmakelists.txt will not even be considered during the build.